src/daft-dsl/src/functions/python/mod.rs

```diff
@@ -2,9 +2,13 @@ mod runtime_py_object;
 mod udf;
 mod udf_runtime_binding;
 
-use std::{collections::HashMap, sync::Arc};
+#[cfg(feature = "python")]
+use std::collections::HashMap;
+use std::sync::Arc;
 
-use common_error::{DaftError, DaftResult};
+#[cfg(feature = "python")]
+use common_error::DaftError;
+use common_error::DaftResult;
```

Comment:
I believe it's because they're only being used by:
- `bind_stateful_udfs`
- `extract_partial_stateful_udf_py`

Both of which are feature-flagged behind the `"python"` flag.